cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -2277,16 +2277,16 @@ Slice::JavaGenerator::getTypeMetadata(const MetadataList& metadata, string& inst
     {
         if (m->directive() == "java:type")
         {
-            string_view arguments = m->arguments();
+            string arguments = m->arguments();
```

Comment:
This is fine since performance is not a concern for this code.

A more exact conversion would be to use `const string&` ... it works too.